tensorflow/core/util/ctc/BUILD

```diff
@@ -59,7 +59,8 @@ tf_cc_tests(
     name = "ctc_beam_search_test",
     size = "small",
     srcs = [
-        "ctc_beam_search_test.cc",
+        "ctc_beam_search_test_float.cc",
```

Comment:
make a single .cc file to test both and use templating within that file.

Reply:
resolved in 51cd5877412f9d67f2ec180c8087ae8d7a059da1